servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -2506,7 +2506,7 @@ RendererCanvasRenderRD::RendererCanvasRenderRD(RendererStorageRD *p_storage) {
 		actions.renames["FRAGCOORD"] = "gl_FragCoord";
 		actions.renames["POINT_COORD"] = "gl_PointCoord";
 
-		actions.renames["LIGHT_POSITION"] = "light_pos";
+		actions.renames["LIGHT_POSITION"] = "light_position";
```

Comment:
For info: It's renamed to conform with the parameter name in the `light_compute` function. Without it using LIGHT_POSITION leads to a shader crash.